styles.css

```diff
@@ -0,0 +1,886 @@
+}
+
+.sub-section-black {
+  background-color: #282b32;
+  color: white;
+  margin-top: -15px;
+}
+
+.sub-section-black,
+.sub-section-white {
+  padding: 1rem;
+}
+
+.program-icon-div {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.program-list h3 {
+  word-break: break-all;
+  width: 30%;
+}
+
+.program-list p {
+  width: 50%;
+  padding: 0.2rem;
+}
+
+.red-color {
+  color: #ff5438 !important;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.line,
+.line-grey {
+  height: 0.2rem;
+  width: 2rem;
+  background-color: #ec5242;
+  margin: 0 0 1.5rem 0;
+}
+
+.line-grey {
+  background-color: rgb(192, 172, 172);
+  height: 0.1rem;
+  width: 1.5rem;
+  margin: 0.5rem 0;
+}
+
+.program-icons {
+  height: 3.5rem;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.program-list div {
+  margin: 0.5rem 0;
+  background-color: #3b3e45;
+  padding: 0.5rem;
+}
+
+.partner-logo {
+  height: 3rem;
+  margin-left: 3rem;
+}
+
+.footer-logo {
+  height: 40px;
+}
+
+.copyrights {
+  font-weight: 700;
+  font-size: 12px;
+}
+
+.cards {
+  margin: 1.5rem 0;
+  padding: 1rem;
+  color: white;
+  border-radius: 5px;
+  display: flex;
+}
+
+.image-container {
+  width: 250px;
+  height: 175px;
+  position: relative;
+}
+
+.speaker-image {
+  width: 178px;
+  height: 177px;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  z-index: 1;
+}
+
+.speaker-details {
+  margin: 1rem 1rem;
+  width: 75%;
+}
+
+.sub-section-text-black {
+  font-size: 14px;
+  color: black;
+  font-weight: 700;
+  line-height: 21px;
+  letter-spacing: -0.05em;
+  padding-top: 0.4rem;
+}
+
+.pad-top {
+  margin-top: 2.5rem;
+}
+
+.about-main-paragraph {
+  margin: 0 8rem 4rem 0;
+  background-color: white;
+}
+
+.contact p {
+  margin-bottom: 0.5rem;
+  text-align: center;
+  font-size: 0.8rem;
+}
+
+.contact a {
+  font-weight: 700;
+  font-size: 0.8rem;
+}
+
+.sub-section-white {
+  background-color: white;
+}
+
+.logo-image {
+  margin: 2rem 0 0 0;
+  border: 1px solid rgb(230, 223, 223);
+  padding: 2rem;
+}
+
+.divider {
+  border-top: 1px solid #e5dfdf;
+  width: 100%;
+}
+
+.warsaw {
+  background:
+    linear-gradient(rgb(240, 6, 6, 0.4), rgb(240, 6, 6, 0.4)),
+    url('images/about_2011_1.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: scroll;
+}
+
+.buenos_aires {
+  background: linear-gradient(rgb(240, 6, 6, 0.4), rgb(240, 6, 6, 0.4)), url('images/about_2013_1.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: scroll;
+}
+
+.simple-cards {
+  padding: 2rem 10vw;
+}
+
+.simple-cards > div {
+  margin: 1rem 0;
+  height: 40vh;
+}
+
+.footer-about {
+  justify-content: space-around;
+  align-items: center;
+  padding-bottom: 3rem;
+  padding-top: 3rem;
+}
+
+/** Style for MOBILE */
+
+@media only screen and (max-width: 767px) {

```

Comment:
- [ ] For the mobile version (**about page**), all the content of your app are not displaying at the exact `width: 375px`, kindly address this. 👍 
<img width="282" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235202643-86bd7a39-7c9c-4e87-a8f7-801d8977c15a.png">

- [ ] For the mobile version (**homepage**), the header and the main program section are overlapping and not displaying as expected. 

<img width="274" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235203537-ec644f75-87a1-484f-a9c0-25aa3e03b63c.png">

Reply:
Done.